app/notices/shop_employer_notices/broker_hired_confirmation_notice.rb

```diff
@@ -0,0 +1,17 @@
+class ShopEmployerNotices::BrokerHiredConfirmationNotice < ShopEmployerNotice
+
+  def deliver
+    build
+    append_data
```

Comment:
For now, we are not using the "append_data" method in this notice but in future, we might need it to append the data related to plan years or enrollments. I can add a comment in the method